src/utils/config.js

```diff
@@ -41,17 +41,12 @@ function loadWallpaper() {
   const el = document.getElementById('desktop');
   if (config.wallpaper) {
     if (
-      // TODO: replace conditions with regex
-      config.wallpaper.startsWith('http') ||
-      config.wallpaper.startsWith('https')
+      config.wallpaper.match(/(http:\/\/)|(https:\/\/)/)
```

Comment:
Can we switch to `test` over `match` its a bit more straight forward although they technically give the same outcome here.

Reply:
switched!